src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = getNumbers(1, totalPages);
+
+  const handleChangePrev = () => (
+    currentPage !== 1
+      ? onPageChange(currentPage - 1)
+      : onPageChange(currentPage)
+  );
+
+  const handleChangeNext = () => (
+    currentPage !== totalPages
+      ? onPageChange(currentPage + 1)
+      : onPageChange(currentPage)
```

Comment:
your button is disable when currentPage === totalPages, so I guess your expression will be never executed, so it's redundant